### ISSUE TITLE
does capacitance values of same finger  in same pression are same?

### ISSUE THREAD

**Issue by webleb** [hankes-garden/capacitive_screen#3]
Hi 
Thanks for this great solution 
i want to ask if same finger give same capacitance values

**Comment by webleb** [comment]
@hankes-garden 
does same finger give same capacitance values for a touch ?

**Comment by Qbart1337** [comment]
yes it does. the range of your finger will match a capacaty value which you can sum up. The real problem are the different divers of the screen on each phone. Can't get a generic solution to have the same values on thisabstraction  layer.

Ges endet: Freitag, 20. September 2019 um 01:23 Uhr
Von: "webleb" <notifications@github.com>
An: hankes-garden/capacitive_screen <capacitive_screen@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: Re: [hankes-garden/capacitive_screen] does capacitance values of same finger in same pression are same? (#3)


@hankes-garden
does same finger give same capacitance values for a touch ?

**Comment by webleb** [comment]
@Qbart1337 
so in conclusion :
same finger on same device have same value 
different finger on same device have different values 
but the problem you cant get same value on different device right ?

**Comment by Qbart1337** [comment]
yes,

yes but only the area is getting bigger (form still remains the same, so not a big issue it can be fixed by area/capacity value)

yes and here it get complicated, it must be reparsed in the application maybe. But than i dont know which all the phones will get me. I had to test a lot of phones to find a proper medium how to filter the raw data.

if ypu find a solution let me know, i struggled there :)

 


Gesendet: Montag, 23. September 2019 um 20:01 Uhr
Von: "webleb" <notifications@github.com>
An: hankes-garden/capacitive_screen <capacitive_screen@noreply.github.com>
Cc: Qbart1337 <Stefan.Aion@web.de>, Mention <mention@noreply.github.com>
Betreff: Re: [hankes-garden/capacitive_screen] does capacitance values of same finger in same pression are same? (#3)


@Qbart1337
so in conclusion :
same finger on same device have same value
different finger on same device have different values
but the problem you cant get same value on different device right ?

**Comment by webleb** [comment]
@Qbart1337 
add me skype:inf.dealer@hotmail.com
for sure we will find a solution 
lets have a conversation in private

**Comment by Qbart1337** [comment]
dont have skype anymore. Switched to discord since years. 
Additionally i did't touch this project for 2 years and i'm pretty stuffed right now. 
Why do you need my help?

**Comment by webleb** [comment]
@Qbart1337
it is possible to make an app that calculate finger capacitance and upload to google play 
?

**Comment by webleb** [comment]
@Qbart1337 
one app can work on all device or we need to get the source code of each device driver ?
